src/json/members/msli.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Michael Li",
+  "pennkey": "msli",
+  "major": "Communications + Data Science",
+  "school": "CAS",
+  "bio": "Hallo! My name is Michael and I like cats even (if they don't like me).",
+  "hometown": "Rochester, NY",
+  "team": "Penn Mobile",
+  "roles": ["Business Developer],
```

Can you change this to "Business Developer"
```suggestion
  "roles": ["Business Developer"],
```